From 906705af9032b6a4b7d1cc771b0f1260801a3ae6 Mon Sep 17 00:00:00 2001
From: Dmitry Shmidt <dimitrysh@google.com>
Date: Wed, 11 Dec 2019 12:59:09 -0800
Subject: [PATCH] YUKAWA: arm64: dts: meson-g12a-sei510: add gpio bluetooth
 interrupt

add gpio irq to support interrupt trigger mode.
(original FROMGIT: arm64: dts: meson-sm1-sei610: add gpio bluetooth interrupt)

Change-Id: Ia95559396a4e163a7e3f05478a5bd8947b1f79f9
Signed-off-by: Dmitry Shmidt <dimitrysh@google.com>
---
 arch/arm64/boot/dts/amlogic/meson-g12a-sei510.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/amlogic/meson-g12a-sei510.dts b/arch/arm64/boot/dts/amlogic/meson-g12a-sei510.dts
index b31502727d4a..4d458913581d 100644
--- a/arch/arm64/boot/dts/amlogic/meson-g12a-sei510.dts
+++ b/arch/arm64/boot/dts/amlogic/meson-g12a-sei510.dts
@@ -615,6 +615,8 @@
 
 	bluetooth {
 		compatible = "brcm,bcm43438-bt";
+		interrupt-parent = <&gpio_intc>;
+		interrupts = <95 IRQ_TYPE_LEVEL_HIGH>;
 		shutdown-gpios = <&gpio GPIOX_17 GPIO_ACTIVE_HIGH>;
 		max-speed = <2000000>;
 		clocks = <&wifi32k>;
